Gems/Atom/Feature/Common/Code/Source/Mesh/MeshFeatureProcessor.cpp

```diff
@@ -37,6 +38,7 @@
 #include <AzCore/Asset/AssetCommon.h>
 #include <AzCore/Name/NameDictionary.h>
 
+#include <Atom/RPI.Public/AssetQuality.h>
```

Comment:
nit: it'd be nice to keep the Atom/RPI.Public includes grouped together for consistency